docs/guide/java_serialization_guide.md

```diff
@@ -83,7 +83,7 @@ public class Example {
   private static final ThreadSafeFury fury = Fury.builder()
     // Allow to deserialize objects unknown types, more flexible
     // but may be insecure if the classes contains malicious code.
-    .requireClassRegistration(true)
+    .requireClassRegistration(false)
```

Comment:
Could we delete comments and keep it as true? And invoke `fury.register(SomeClass)`  at the initialization

Reply:
done. please help review again